mjpc/test/simulation.h

```diff
@@ -0,0 +1,61 @@
+// Copyright 2022 DeepMind Technologies Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef MJPC_TEST_SIMULATION_H_
+#define MJPC_TEST_SIMULATION_H_
+
+#include <mujoco/mujoco.h>
+
+#include <functional>
+
+#include "mjpc/estimators/trajectory.h"
+
+namespace mjpc {
+
+// simulation
+class Simulation {
+ public:
+  // constructor
```

Comment:
Comment on taking ownership of mjModel (or better, use UniqueMjModel UniqueMjData to manage these things).

Reply:
Changed.

Reply:
I don't see the change. Have you pushed it?